ci/debian-10/Dockerfile

```diff
@@ -2,9 +2,11 @@ FROM debian:10
 
 ENV DEBIAN_FRONTEND="noninteractive" TZ="America/Los_Angeles"
 
+ENV CMAKE_DIR "/opt/cmake"
+ENV CMAKE_VERSION "3.19.1"
```

Comment:
I vaguely remember conversations about this, but are we bumping the minimum cmake required to build Zeek in general?

Reply:
Since the decision was to enable Spicy support by default we will need to bump the required CMake version. I'll update `cmake_minimum_required` in `CMakeLists.txt` for that.